packages/tools/src/tools/annotation/KeyImageTool.ts

```diff
@@ -56,6 +68,11 @@ class KeyImageTool extends AnnotationTool {
         changeTextCallback,
         canvasPosition: [10, 10],
         canvasSize: 10,
+        handleRadius: '6',
+        /** If true, this selects the entire series/display set */
+        seriesLevel: false,
```

Comment:
I don't see any logic for `seriesLevel` being used. It seems like it's just acting as storage. Can't we just remove it and have the OHIF outside the application handle putting everything in the config?

Reply:
The problem is that the SR representation needs to know this information on store/retrieve.  I've added logic that depends on this, although it isn't quite complete yet.